runner/local/simple.go

```diff
@@ -88,47 +79,57 @@ func (r *simpleRunner) StatusAll() ([]runner.ProcessStatus, error) {
 	return statuses, nil
 }
 
-func (r *simpleRunner) Abort(run runner.RunId) (runner.ProcessStatus, error) {
-	return r.updateStatus(runner.AbortStatus(run))
+func (r *simpleRunner) Abort(runId runner.RunId) (runner.ProcessStatus, error) {
```

Comment:
Seems like the Go way is to prefer "ID" so I will go with that for files touched in this review.  If we want to revert files to sue that I can make a ticket, or we can start requesting changes to other reviews as it comes up.